cmd/removeProcessor.go

```diff
@@ -35,6 +35,7 @@ func newRemoveTransferProcessor(cca *CookedCopyCmdArgs, numOfTransfersPerPart in
 		SourceRoot:          cca.Source.CloneWithConsolidatedSeparators(), // TODO: why do we consolidate here, but not in "copy"? Is it needed in both places or neither? Or is copy just covering the same need differently?
 		CredentialInfo:      cca.credentialInfo,
 		ForceIfReadOnly:     cca.ForceIfReadOnly,
+		BlobFSRecursiveDelete: cca.Recursive,
```

Comment:
nit, do we want to call this BlobFs? What if this becomes relevant to another service?

Reply:
We can just rename it later if it does. Not breaking. I don't think it will.